templates/registration/logged_out.html

```diff
@@ -0,0 +1,6 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <p> You have logged out!</p>
+  <p>Go to <a href="#">home</a> page or <a href="{% url 'login' %}">login again</a></p>
```

Comment:
for home page you may use 
```suggestion
  <p>Go to <a href="{% taxi:index %}">home</a> page or <a href="{% url 'login' %}">login again</a></p>
```

Reply:
Thanks for you suggestions